torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -419,21 +419,21 @@ def aten_new_full_dtype(self: TTensor, size: INT64, fill_value: TTensor, dtype:
 
     """
 
-    def __init__(self, op_schema: onnx.defs.OpSchema):
+    def __init__(self, onnxfunction: "onnxscript.OnnxFunction"):
         """Initialize the OpSchemaWrapper.
```

Comment:
This comment is also for the class name and class docstring above that github does not let me add comments. 
Better to update all names and docstrings, sry!

But I believe this is right and good change. Crux is in order to do input match, we also require onnxfunction param **ordering**. Such info is exclusively in `OnnxFunction.param_schemas()`, and not in `OpSchema`.

Reply:
Use `_OnnxSchemaChecker`